Tests/BuildTests/BuildPlanTests.swift

```diff
@@ -3431,111 +3431,6 @@ final class BuildPlanTests: XCTestCase {
         XCTAssertMatch(executablePathExtension, "exe")
     }
 
-    func testWASITarget() throws {
```

Comment:
Moved to `Tests/BuildTests/CrossCompilationBuildPlanTests.swift` to make `Tests/BuildTests/BuildPlanTests.swift` smaller and increase parallelization during testing, as separate `XCTestCase` types can be parallelized, while multiple functions in `XCTestCase` can't.